tcf_website/templates/search/search.html

```diff
@@ -41,8 +50,13 @@ <h2>
         <div id="results" class="accordion">
 
           <!-- Course results -->
+          {% if courses_first %}
           <div id="courses" class="row mb-5 collapse show"
```

Comment:
Similar thing - instead of wrapping the entire block in a conditional, you can just wrap the `show` attribute.